src/figuresClasses.ts

```diff
@@ -1,19 +1,70 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
According to the task, we have defined certain types of shapes that can be, and by specifying a general type of `string`, I can write anything there and it will work. The same for color:
<img width="491" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/54860641/1cdd156a-a9fd-41dd-a033-fe50ed222fe3">